src/components/NavBar.scss

```diff
@@ -1,35 +1,60 @@
-.Nav {
-	background-color: var(--color-bg);
-	border-top: 1px solid var(--color-border);
-	bottom: 0;
-	display: flex;
-	flex-direction: row;
-	padding-bottom: max(env(safe-area-inset-bottom), 1rem);
-	padding-top: 1rem;
-	place-content: center;
-	position: fixed;
-	width: 100%;
-
-	&__container {
-		display: flex;
-		flex-direction: row;
-		justify-content: space-evenly;
-		width: min(72ch, 100%);
+@import "../styles/variables.scss";
+
+.navbar {
+	.navbar-brand {
+		font-size: 3.5rem !important;
+
+		text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.4);
+
+		@media (max-width: 430px) {
```

Comment:
Wow! So cool! I didin't know you could include media queries directly into the css custom class definition itself!

Reply:
I don't remember where I saw it but I was like let me try that lol